pyccel/codegen/printing/fcode.py

```diff
@@ -369,7 +369,7 @@ def _print_Program(self, expr):
     def _print_Import(self, expr):
 
         source = ''
-        if str(expr.source) in pyccel_builtin_import_registery:
+        if str(expr.source) in pyccel_builtin_import_registery or expr.ignore:
```

Comment:
print_import print also additional import and in C we handle some of the math function with additional import, and they are printed in C code even if they exist in pyccel_builtin_import_register.

I may propose this solution:
1 - drop the property ignore and use a condition in _builtin_import_registery and not in additional import_

Reply:
That sounds like a good solution. Presumably, currently in Fortran, we cannot import a module which shares a name with something in `pyccel_builtin_import_registery`?
That said, we do not have this condition in C and we don't have any problems. Presumably because the import was ignored at the semantic stage. [Codacy confirms that this condition is not used in the Fortran code](https://app.codacy.com/gh/pyccel/pyccel/file/51897655523/coverage?bid=15415567&fileBranchId=15415567#l373).

I think the simpler solution would therefore be to change this condition to `if expr.ignore`.
What do you think?